ext/curl/multi.c

```diff
@@ -548,9 +540,9 @@ static void curl_multi_free_obj(zend_object *object)
 
 	curl_multi_cleanup(mh->multi);
 	zend_llist_clean(&mh->easyh);
-	if (mh->handlers.server_push) {
-		zval_ptr_dtor(&mh->handlers.server_push->func_name);
-		efree(mh->handlers.server_push);
+
+	if (ZEND_FCC_INITIALIZED(mh->handlers.server_push)) {
+		zend_fcc_dtor(&mh->handlers.server_push);
```

Comment:
In other places you set it then to the empty fcc, did you forget to do that here?